code/obj/item/clothing/chameleon_js.dm

```diff
@@ -269,3 +272,1659 @@
+		name = "robotics backpack"
+		desc = "A thick, wearable container made of synthetic fibers, able to carry a number of objects monochromaticly on the back of roboticists."
+		icon_state = "bp_robotics"
+		item_state = "bp_robotics"
+
+	robotics_satchel
+		name = "robotics satchel"
+		desc = "A thick, wearable container made of synthetic fibers, able to carry a number of objects monochromaticly on the shoulder of roboticists."
+		icon_state = "satchel_robotics"
+		item_state = "satchel_robotics"
+
+	genetics
+		name = "genetics backpack"
+		desc = "A thick, wearable container made of synthetic fibers, able to carry a number of objects safely on the back of geneticists."
+		icon_state = "bp_genetics"
+		item_state = "bp_genetics"
+
+	genetics_satchel
+		name = "genetics satchel"
+		desc = "A thick, wearable container made of synthetic fibers, able to carry a number of objects safely on the shoulder of geneticists."
+		icon_state = "satchel_genetics"
+		item_state = "satchel_genetics"
+
+	medic
+		name = "medic's backpack"
+		desc = "A thick, wearable container made of synthetic fibers, able to carry a number of objects comfortably on a Medical Doctor's back."
+		icon_state = "bp_medic"
+		item_state = "bp-medic"
+
+	medic_satchel
+		name = "medic's satchel"
+		desc = "A thick, wearable container made of synthetic fibers, able to carry a number of objects comfortably on a Medical Doctor's shoulder."
+		icon_state = "satchel_medic"
+		item_state = "satchel_medic"
+
+	captain
+		name = "Captain's Backpack"
+		desc = "A fancy designer bag made out of space snake leather and encrusted with plastic expertly made to look like gold."
+		icon_state = "capbackpack"
+		item_state = "capbackpack"
+
+	captain_satchel
+		name = "Captain's Satchel"
+		desc = "A fancy designer bag made out of space snake leather and encrusted with plastic expertly made to look like gold."
+		icon_state = "capsatchel"
+		item_state = "capsatchel"
+
+/obj/item/remote/chameleon
+	name = "chameleon outfit remote"
+	desc = "A remote control that allows you to change an entire set of chameleon clothes, all at once."
+	icon = 'icons/obj/porters.dmi'
+	icon_state = "remote"
+	item_state = "electronic"
+	w_class = W_CLASS_SMALL
+
+	var/obj/item/storage/backpack/chameleon/connected_backpack = null
+	var/obj/item/clothing/under/chameleon/connected_jumpsuit = null
+	var/obj/item/clothing/head/chameleon/connected_hat = null
+	var/obj/item/clothing/suit/chameleon/connected_suit = null
+	var/obj/item/clothing/glasses/chameleon/connected_glasses = null
+	var/obj/item/clothing/shoes/chameleon/connected_shoes = null
+	var/obj/item/storage/belt/chameleon/connected_belt = null
+	var/obj/item/clothing/gloves/chameleon/connected_gloves = null
+	var/list/outfit_choices = list()
+
+	New()
+		..()
+		for(var/U in (typesof(/datum/chameleon_outfit_choices)))
+			var/datum/chameleon_outfit_choices/P = new U
+			src.outfit_choices += P
+		return
+
+	attack_self()
+		if (isliving(usr))
+			set src in usr
```

Comment:
```suggestion
```
This doesn't do anything, as it isn't in a verb. Also if someone's hitting this with itself they have to be holding it anyways (i hope!)